docs/user/sbt.rst

```diff
@@ -12,21 +12,21 @@ The easiest way to make a fresh project is to use our official gitter8 template:
 
     sbt new scala-native/scala-native.g8
 
-This includes:
+This includes the following files with the proper version entered (denoted by <version>):
 
 * ``project/plugins.sbt`` to add a plugin dependency::
 
-    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.7")
+    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "<scala-native-version>")
 
 * ``project/build.properties`` to specify the sbt version::
 
-    sbt.version = 0.13.17
+    sbt.version = <sbt-version>
 
 * ``build.sbt`` to enable the plugin and specify Scala version::
 
     enablePlugins(ScalaNativePlugin)
 
-    scalaVersion := "2.11.12"
+    scalaVersion := "<scala-version>"
```

Comment:
After having another look at this, I'm convinced this is guaranteed to confuse people. Previous thing was just copy-paste and run no questions asked and no thinking required. Suggested change leaves no info on what combination of versions is known to work/recommended for use with Scala Native. Concrete versions are much more clear, even if it have to update them manually at every release.

Reply:
Ok, this makes sense. When the `g8` is updated then the versions in this document need to be updated. You can't really change the `g8` until after the release or right when you make the release.

Reply:
We typically change g8 as soon as release syncs to sonatype, but before announcing publicly, so people don't really perceive the gap between the two. I don't think it's an issue we should worry about.

Reply:
I totally see the logic, I was trying to be fancy to avoid making changes but it could certainly be too confusing especially for new users.